### PR TITLE
Fix/unify icons

### PR DESCRIPTION
**Pull request by RajVerma97**
Unify icons on the stats card #589

Screenshot 2024-09-30 at 1 25 19 AM

### CODE REVIEW

**Comment by petrvecera** [comment]
Awesome! :)

Can you please try to rebase with master? Or merge master into this so we resolve the conflicts 👍

File: components/unit-cards/unit-squad-card.tsx [hunk 1]
```diff
@@ -43,6 +43,11 @@ type UnitSquadInput = {
   };
 };
 
+const unfiformIconSize = {
+  width: 24,
+  height: 24,
```
Comment (petrvecera) [inline components/unit-cards/unit-squad-card.tsx:48]: Let's make it little bit smaller 20 / or 22.

Comment (RajVerma97) [reply components/unit-cards/unit-squad-card.tsx]: done

**Review by petrvecera — APPROVED** [review]
Nice, it looks great! I just commited a fix for the CSS file, I modified it little bit in the master to work also on Mobile view. I forgot to mention that in the previous task.

**Pull request merged**